test/sslapitest.c

```diff
@@ -799,11 +799,17 @@ static int execute_test_large_message(const SSL_METHOD *smeth,
 
     if (!TEST_ptr(certbio = BIO_new_file(cert, "r")))
         goto end;
-    chaincert = PEM_read_bio_X509(certbio, NULL, NULL, NULL);
+
+    chaincert = X509_new_with_libctx(libctx, NULL);
+    if (chaincert == NULL) {
+        SSLerr(0, ERR_R_MALLOC_FAILURE);
```

Comment:
Err....`SSLerr`? In a test? Should be `TEST_ptr` - which you seem to have removed below?